src/screens/BottomNavBar.js

```diff
@@ -27,7 +28,7 @@ export default function BottomNavBar({ navigation }) {
                         if (rn === 'Home') {
                             iconName = focused ? 'ios-home' : 'ios-home-outline'
                             return <Ionicons name={iconName} size={45} color={color} />
-                        } else if (rn === 'Album') {
+                        } else if ((rn === 'AlbumNavigator')) {
```

Comment:
El nombre "rn" es muy poco intuitivo, recomiendo cambiar a algo mas intuitivo
```suggestion
                        } else if ((routeName === 'AlbumNavigator')) {
```